snuba/datasets/storages/errors_common.py

```diff
@@ -143,11 +142,10 @@
     ),
     MappingColumnPromoter(mapping_specs={"tags": promoted_tag_columns}),
     UserColumnProcessor(),
-    EventIdColumnProcessor(),
+    BetterUUIDColumnProcessor({"event_id"}),
```

Comment:
`BetterUUIDColumnProcessor` ?  What about calling this UUIDColumnProcessor and renaming the other one as `legacy` ?

Reply:
Definitely planning to call this UUIDColumnProcessor eventually. My plan is to rename when turning it on everywhere and save myself one rename hop.